### DISCUSSION TITLE
Make Flux components usable with AlpineJS

### DISCUSSION THREAD

**Discussion by pghiuzan-phished** [livewire/flux#860] [Feature Requests]
We ran into a use-case where we need to render Flux components in an `x-for` Alpine loop. Components like `<flux:option />` (and others) are not behaving properly when used in such contexts. It would be great if this problem was addressed.

For context, this issue is derived from https://github.com/livewire/flux/issues/740, which purpose was to bring up this problem but ended up focused on the underlying performance issue.

**Comment by joffreypersia** [comment]
Hi, 
You can not use an Alpine x-for (javascript, so client-side rendering) with FluxUI blade component (that are server side components). One way to loop through blade components is to use a @foreach (so it is server side).
